test/unit/components/SideBar/LastBlockTime.spec.js

```diff
@@ -23,7 +23,8 @@ const Wrapper = ({ lastBlockTimestamp, setTimeout, clearTimeout }) => {
       messages={locale.messages}
       formats={locale.formats}
       defaultFormats={defaultFormats}
-      key={locale.key}>
+      key={locale.key}
+    >
```

Comment:
hmm, didnt mean to change this. Maybe I did prettify incorrectly. Should just be 'yarn prettify'?

Reply:
Probably ok now.